src/rws_poco_client.cpp

```diff
@@ -451,6 +462,24 @@ POCOClient::POCOResult POCOClient::webSocketRecieveFrame()
   return result;
 }
 
+void POCOClient::webSocketClose() {
+  // Make sure nobody is connecting while we're closing.
+  ScopedLock<Mutex> connect_lock(websocket_connect_mutex_);
+
+  // Make sure there is actually a connection to close.
+  if (!webSocketExist()) {
```

Comment:
```suggestion
  if (!webSocketExist())
  {
```
Curly bracket on new line to be consistent with the rest of the code base.